### PR TITLE
[frio] Formatbar in new Post and Share Button for Preview Post

### PR DESCRIPTION
**Pull request by JonnyTischbein**
To Issue #3870 

Change:

 - Add a share button in an own div with own ID
 - in frio js: make div visible when preview post (Needed own div because main view js replace all preview-div content)
 - Add Formatbar to NewPost Form
 - Add `insertFormattingToPost()` (`insertFormatting()` only works for comments)

### CODE REVIEW

File: view/theme/frio/templates/jot.tpl [hunk 1]
```diff
@@ -118,6 +122,13 @@
 				</div>
 
 				<div id="jot-preview-content" class="minimize" aria-labelledby="jot-preview-lnk" role="tabpanel" aria-hidden="true"></div>
+
+				<div id="jot-preview-share" class="minimize" aria-labelledby="jot-preview-lnk" role="tabpanel" aria-hidden="true">
+					<ul id="profile-jot-submit-wrapper" class="jothidden nav nav-pills">
```
Comment (rabuzarus) [inline view/theme/frio/templates/jot.tpl]: The id "profile-jot-submit-wrapper" is already in us but id's should be unique. So you should chose another id name

File: view/theme/frio/templates/jot.tpl [hunk 2]
```diff
@@ -118,6 +122,13 @@
...
+						<li role="presentation" class="pull-right"><button class="btn btn-primary" type="submit" id="profile-jot-submit" name="submit" ><i class="fa fa-slideshare fa-fw" aria-hidden="true"></i> {{$share}}</button></li>
```
Comment (rabuzarus) [inline view/theme/frio/templates/jot.tpl]: The id "profile-jot-submit" is already in us but id's should be unique. So you should chose another id name

**Comment by JonnyTischbein** [comment]
Do you mean smaller screen that this ?
screenshot

What about a second row for the formating bar ?

**Comment by JonnyTischbein** [comment]
I do understand it.

But I do think that a format bar is especially on mobile devices very useful.

What would you think of a switch button for booth symbol bars. Booth have the same amount of symbols.

**Comment by rabuzarus** [comment]
> But I do think that a format bar is especially on mobile devices very useful

Really?
screenshot_20180921-144801

I don't want to say that the formatting buttons should be disabled in general for mobile phones but if the buttons don't fit in one row the screen will be to small to make use of it - so we can disable it for very small screens (class="hidden-xs") to have a slim gui

**Comment by JonnyTischbein** [comment]
Ok, I will add it.

I see the problem but imo it is still better than writing bbcode on mobile phone.

**Comment by rabuzarus** [comment]
> Ok, I will add it.

Thank you

> I see the problem but imo it is still better than writing bbcode on mobile phone.

Yes but the user (for example I) should also recognize that a mobile phone (especially a phone with a very small display) is just a phone and no desktop computer. So maybe the user should limit his/her usage to write text instead of formatting text. I wouldn't write my exams on a phone :-)

**Pull request merged**